### ISSUE TITLE
Use default port for DNS resolver according to given protocol

### ISSUE THREAD

**Issue by northys** [safing/portmaster#147]
**Checklist**:

- [x] I'm using the official portmaster release (i.e no custom builds)
- [x] I searched for similar/existing issues first.
- [x] My issue is not mentioned in the Known Issues section of my [OS](https://github.com/safing/portmaster/wiki)

**What happened**:

Trying to set `dns://8.8.8.8` but I wasn't successful.

**What you expected to happen**:

The default port is used automatically, 53 in this case.

Portmaster automatically uses default port for given protocol when no port specified.

**How to reproduce it (as minimally and precisely as possible)**:

1) ubuntu 20.04
2) latest portmaster
3) set DNS servers to `dns://8.8.8.8`
4) DNS doesn't work because the port `53` is not used automatically

**Anything else we need to know?**:

**Environment**:

Portmaster Version: 

```
Name: Portmaster
Version: 0.5.4
Commit: tags/v0.5.4-0-g6093c5b8476e488b63d3a735b2ad178b7625114b
Build Options: main.go
Build User: user
Build Host: docker
Build Date: 19.08.2020
Build Source: https://github.com/safing/portmaster
```

Operating System:
- [ ] Windows 7
- [ ] Windows 8/8.1
- [ ] Windows 10
- [x] Linux

```
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal
```

**Comment by dhaavi** [comment]
Thanks, that's a good idea to add support for default ports! I will add this to our development plan.

**Comment by dhaavi** [comment]
Fix is released in [v0.5.5](https://github.com/safing/portmaster/releases/tag/v0.5.5). Please report any further issues.
